frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -74,13 +78,38 @@ const MainLayoutContentContainer = styled('div')(({ theme }) => ({
     zIndex: 200,
 }));
 
+const SegmentsSplash: React.FC = () => {
+    const { value: localStorageState, setValue: setLocalStorageState } =
+        createLocalStorage('OssSegmentsSplashScreen:v1', { shown: false });
+
+    const [showSegmentSplash, setShowSegmentSplash] = React.useState(true);
```

Comment:
why do we need both local storage and memory state? can't we solve it with only local storage state?

Reply:
Potentially. I can try it out. Just afraid that we'll end up in a sitch where it doesn't close. But good point 💁🏼

Reply:
Yeah, don't know if I'm doing something wrong, but that just leads to me not being able to close it. Seems innocuous enough to leave it in 🤷🏼

Reply:
ok, so let's keep it as it is :)

Reply:
Yeah, I also have the feeling that this could potentially be slightly simplified. Since I think it would be cool to see more announcements like these in the future, perhaps it's something we could do for a reusable component in a later PR.

Reply:
I think that what's going on is that it doesn't re-render when you update localStorage, so even if you set the localStorage key, that's not picked up until the re-render. Hence the need for an extra variable. 

But if you have suggestions on how to simplify it, I'd be happy to listen 😄 

As for more announcements like these in the future, yeah, maybe. I'm personally not a big fan of in-your-face updates like this, but it _does_ get the point across, I guess. But I suppose it's not up to me to decide for all of us 💁🏼

Reply:
I think this is totally fine for now 🙂 

Yeah, understandable. Maybe a more out-of-the-way message banner could be a better option in the future.